src/components/TooltipWithTouch.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box, Tooltip, type TooltipProps } from '@chakra-ui/react'
+import { type PropsWithChildren, useCallback, useState } from 'react'
+
+type TooltipWithTouchProps = TooltipProps & PropsWithChildren
+
+export const TooltipWithTouch: React.FC<TooltipWithTouchProps> = ({
+  children,
+  ...restToolTipProps
```

Comment:
can memoize this to prevent the children re-rendering like crazy

Reply:
we can remove the spread here